client/src/components/ratings-and-reviews/add-review-form/AddReviewForm.jsx

```diff
@@ -0,0 +1,143 @@
+    axios.post('/reviews', reviewObj)
+      .then((result) => {
+        console.log(result.data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+  return (
+    <div className="modal">
+      <form id="review-form">
+        <h1>Write your review</h1>
+        <div className="rating selection">
+          <p>Overall Rating</p>
+          <div className="dynamic-star">
+            <span id={5} onClick={starHandler}>&#9733;</span>
+            <span id={4} onClick={starHandler}>&#9733;</span>
+            <span id={3} onClick={starHandler}>&#9733;</span>
+            <span id={2} onClick={starHandler}>&#9733;</span>
+            <span id={1} onClick={starHandler}>&#9733;</span>
+          </div>
+        </div>
+        <fieldset className="recommend selection">
+          <legend>Do you recommend this product?</legend>
+          <div>
+            {/* eslint "react/self-closing-comp":0 */}
+            <label htmlFor="yes">
+              Yes
+              <input type="radio" name="recommend" value onClick={inputHandler}></input>
+            </label>
+          </div>
+          <div>
+            <label htmlFor="no">
+              No
+              <input type="radio" name="recommend" value={false} onClick={inputHandler}></input>
+            </label>
+          </div>
+        </fieldset>
+        <div className="characters selection">
+          <p>Characteristics Rating</p>
+          {Object.keys(characteristics).map((key) => (
+            <CharacteristicSelection characteristic={key} key={key} inputHandler={inputHandler} />
+          ))}
+        </div>
+        <div className="summary input">
+          <p>
+            Review Summary
+            <small> (60 characters max)</small>
+          </p>
+          <textarea name="summary" maxLength={60} placeholder="Best purchase ever!" rows={3} value={reviewObj.summary} onChange={inputHandler}></textarea>
+        </div>
+        <div className="details input">
+          <p>
+            Review Details
+            <small> (1000 characters max)</small>
+          </p>
+          <textarea
+            name="body"
+            maxLength={1000}
+            minLength={50}
+            placeholder="Why did you like the product or not?"
+            rows={20}
+            cols={50}
+            value={reviewObj.body}
+            onChange={inputHandler}
+          >
+          </textarea>
+        </div>
+        <PhotoUpload inputHandler={inputHandler} />
+        <div className="nickname input">
+          <p>
+            What is your nickname?
+            <small> (60 characters max)</small>
+          </p>
+          <textarea name="name" maxLength={60} placeholder="jackson11" rows={3} value={reviewObj.name} onChange={inputHandler}></textarea>
+          <small>For privacy reasons, do not use your full name or email address</small>
+        </div>
+        <div className="email input">
+          <p>
+            What is your nickname?
```

Comment:
Do you mean to ask "What is your email" here?

Reply:
yes, thanks for catching that